test/commands/trigger_test.exs

```diff
@@ -1,13 +1,15 @@
 defmodule Cog.Test.Commands.TriggerTest do
-  use Cog.CommandCase, command_module: Cog.Commands.Trigger
+  use Cog.CommandCase, command_tag: :trigger
```

Comment:
Yeah, that would also work. We can reorganize those bits. Is that something that should happen in these PRs or as separate work?